app/komponenter/veilederpanel/veilederpanel.module.css

```diff
@@ -8,15 +8,15 @@
   }
 }
 
-.tekstInnholdMellomrom {
+.bunnMargin {
   margin-bottom: 1.2rem;
 }
 
-.poster {
+.posterKonteiner {
   text-align: left;
   width: 100%;
 }
 
-.poster ul {
+.posterKonteiner ul {
   margin: 0 auto 2rem auto;
 }
```

Comment:
Hvis man bruker samme klasse for alle veilederpanel kan denne også bare bli `veilederpanel ul` fordi den brukes kun hvis det finnes `ul`

Reply:
Fikset i commit 7170681